dataprep/eda/correlation/compute/nullivariate.py

```diff
@@ -31,6 +39,134 @@ def _calc_nullivariate(
     # So we do them in pandas
 
     (corrs,) = dask.compute(corrs)
+    pearson = corrs[CorrelationMethod.Pearson]
+    spearman = corrs[CorrelationMethod.Spearman]
+    kendalltau = corrs[CorrelationMethod.KendallTau]
+
```

Comment:
Seems the processing of each correlation type is the same. Could we write a function to reduce the duplicate code? Besides, the variable name is not self-explained. Would be better to use full name.